plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnector.java

```diff
@@ -42,7 +42,7 @@
  * This is the same for any test cases that were creating tables with duplicate rows,
  * so some test cases are overridden from the base class and slightly modified to add an additional UUID column.
  */
-public class TestAccumuloConnectorTest
+public class TestAccumuloConnector
```

Comment:
I guess `BaseConnectorTest#ensureTestNamingConvention` will fail after this change.